### PR TITLE
HPCC-17132 DOCS:ConfigMgr can use IP or Hostnames

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17132 DOCS:ConfigMgr can use IP or Hostnames
Document that we can use IP addresses of the host names
in configuring the HPCC.

@rpastrana please review
@JamesDeFabia please review

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [x] Scalability
  - [x] Performance
  - [x] Security
  - [x] Thread-safety
  - [x] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:

Tested by building the changed documents. All built successfully.

### CODE REVIEW

**Review by rpastrana — CHANGES_REQUESTED** [review]
@g-pan some minor comments

**Comment by richardkchapman** [comment]
@g-pan Back to you

File: docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 1]
```diff
@@ -912,9 +912,6 @@
             <para>Press the <emphasis role="bold">Next</emphasis>
             button.</para>
 
-            <para>Alternatively, you could find the IP addresses using Auto
```
Comment (g-pan) [inline docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: Addressed in https://github.com/hpcc-systems/HPCC-Platform/pull/9703
only not merged yet, Removed this bit to ensure the text does not get reintroduced.

**Comment by rpastrana** [comment]
@gpan squash these commits, and update the "add hardware" screen documentation.

**Comment by richardkchapman** [comment]
@g-pan

**Comment by g-pan** [comment]
Comments addressed. Squashed and rebased.

**Comment by g-pan** [comment]
@rpastrana please review

**Comment by rpastrana** [comment]
@g-pan changes look fine. I did not confirm all necessary sections were updated.

**Pull request merged**